index.html

```diff
@@ -81,8 +81,8 @@ <h2 class="txt-center">
       <i class="ion-android-sync spinner fontSize30 spinner-with-logo-icon"></i>
     </div>
     <div class="width100 positionTop">
-      <a class="btn btn-txt rowTop10 marginRight10 pull-right js-indexReload"><i class="ion-android-sync"></i> Reload</a>
-      <a class="btn btn-txt rowTop10 marginRight10 pull-right" href="#home"><i class="ion-home"></i> Load Index</a>
+      <a class="btn btn-txt rowTop10 marginRight10 pull-right js-indexReload"><i class="ion-android-sync"></i> <t>Reload</t></a>
```

Comment:
Can this be done with a span and a class, like js-translate? I know it works fine like this, but I don't like putting in non-standard tags.